modules/devguide/examples/StartUsingCloud.java

```diff
@@ -0,0 +1,85 @@
+// tag::cloud-connect[]
+// tag::imports[]
+import static com.couchbase.client.java.query.QueryOptions.queryOptions;
+
+import com.couchbase.client.core.env.SecurityConfig;
+import com.couchbase.client.java.*;
+import com.couchbase.client.java.env.ClusterEnvironment;
+import com.couchbase.client.java.kv.*;
+import com.couchbase.client.java.json.*;
+import com.couchbase.client.java.query.*;
+
+import java.time.Duration;
+
+import static com.couchbase.client.java.query.QueryOptions.queryOptions;
+// end::imports[]
+
+
+public class StartUsingCloud {
+  // tag::connect-info[]
+  // Update these variables to point to your Couchbase Capella instance and credentials.
+  static String connectionString = "cb.5lql7zxho580-tyt.cloud.couchbase.com";
+  static String username = "username";
+  static String password = "Password!123";
+  static String bucketName = "travel-sample";
+  // end::connect-info[]
+
+  public static void main(String... args) {
+    // tag::connect-string[]
+    // Simple connection with TLS enabled.
+    Cluster cluster = Cluster.connect("couchbases://" + connectionString, username, password);
+    // end::connect-string[]
+    cluster.disconnect();
+
+    // tag::connect-env[]
+    // Custom environment connection with TLS enabled.
+    ClusterEnvironment env = ClusterEnvironment.builder()
+            .securityConfig(SecurityConfig.enableTls(true))
+            // Sets a pre-configured profile called "wan-development" to help avoid latency issues
+            // when accessing Capella from a different Wide Area Network
+            // or Availability Zone (e.g. your laptop).
+            .applyProfile("wan-development")
+            .build();
+
+    cluster = Cluster.connect(
+            connectionString,
+            ClusterOptions.clusterOptions(username, password).environment(env)
+    );
```

Comment:
@daschl or @mikereiche Please let me know if this is the correct way to set a configuration profile, is there anything else I should add?

Reply:
Looks like the *Volatile* question I had was answered in the SDK meeting - I will update the PR.

Reply:
Hi Maria!

That code snippet would work, but would require the user to shut down the environment themselves, and would ignore the connection string scheme and parameters. In general, users should avoid the version of  `ClusterOptions.env` that takes a pre-build environment; instead they should the version that takes a callback to configure an environment builder, so the resulting ClusterEnvironment is owned and managed by the Cluster, and honors the connection string scheme and parameters.

Assuming `connectionString` starts with "couchbase**s**://", here's what I'd recommend:

```java
cluster = Cluster.connect(
    connectionString,
    ClusterOptions.clusterOptions(username, password).environment(env -> {
      // Sets a pre-configured profile called "wan-development" to help avoid latency issues
      // when accessing Capella from a different Wide Area Network
      // or Availability Zone (e.g. your laptop).
      env.applyProfile("wan-development");
    })
);
```

By the way, great explanation of when/why to use this profile!

Thanks,
David